### PR TITLE
Keep consistency of buttons  on host dashboard

### PR DESCRIPTION
**Pull request by JoyLubega**
Resolve: opencollective/opencollective/issues/2570

Screen Shot 2019-11-27 at 3 01 37 PM

### CODE REVIEW

File: components/expenses/Expense.js [hunk 1]
```diff
@@ -461,7 +461,11 @@ class Expense extends React.Component {
                       />
                     )}
                     {canPay && (
-                      <StyledButton mr={2} onClick={() => this.setState({ showUnapproveModal: true })}>
+                      <StyledButton
+                        mr={2}
+                        buttonStyle="dark"
```
Comment (Betree) [inline components/expenses/Expense.js]: We should keep the default state for this one, unless there's a good rational for using a dark style

File: components/expenses/MarkExpenseAsUnpaidBtn.js [hunk 2]
```diff
@@ -54,7 +54,11 @@ const MarkExpenseAsUnpaidBtn = ({ id, markExpenseAsUnpaid, refetch }) => {
           </StyledButton>
         </Fragment>
       ) : (
-        <StyledButton onClick={() => setState({ ...state, showProcessorFeeConfirmation: true })} mt={2}>
+        <StyledButton
+          buttonStyle="dark"
```
Comment (Betree) [inline components/expenses/MarkExpenseAsUnpaidBtn.js]: Same here, we should keep the default state for this one unless there's a good rational for using a dark style

File: package.json [hunk 3]
```diff
@@ -50,7 +50,7 @@
     "glob": "7.1.6",
     "graphql": "14.5.8",
     "graphql-tag": "2.10.1",
-    "helmet": "3.21.2",
+    "helmet": "^3.21.2",
```
Comment (Betree) [inline package.json]: Should not have any change, especially because we want to use fixed dependencies

File: lib/theme/index.js [hunk 4]
```diff
@@ -80,6 +80,64 @@ export const generateTheme = ({
       },
     },
 
+    danger: {
+      backgroundColor: colors.red[600],
```
Comment (Betree) [inline lib/theme/index.js]: In the [design system](https://www.figma.com/file/N4Xbl652BhzOutXmzhcrLGch/DS-06-%2F-Atoms?node-id=14%3A1) the following colors have been defined for the `danger` button:

```suggestion
      backgroundColor: colors.red[500],
```

File: lib/theme/index.js [hunk 5]
```diff
@@ -80,6 +80,64 @@ export const generateTheme = ({
...
+      borderColor: colors.red[600],
```
Comment (Betree) [inline lib/theme/index.js]:
```suggestion
      borderColor: colors.red[500],
```

File: lib/theme/index.js [hunk 6]
```diff
@@ -80,6 +80,64 @@ export const generateTheme = ({
...
+      color: 'white',
+
+      '&:hover': {
+        backgroundColor: colors.red[700],
```
Comment (Betree) [inline lib/theme/index.js]:
```suggestion
        backgroundColor: colors.red[400],
```

File: lib/theme/index.js [hunk 7]
```diff
@@ -80,6 +80,64 @@ export const generateTheme = ({
...
+        borderColor: colors.red[700],
```
Comment (Betree) [inline lib/theme/index.js]:
```suggestion
        borderColor: colors.red[400],
```

File: lib/theme/index.js [hunk 8]
```diff
@@ -80,6 +80,64 @@ export const generateTheme = ({
...
+        color: 'white',
+      },
+
+      '&:focus': {
+        backgroundColor: colors.red[500],
+        borderColor: colors.red[700],
+      },
+
+      '&:active': {
+        backgroundColor: colors.red[800],
```
Comment (Betree) [inline lib/theme/index.js]:
```suggestion
        backgroundColor: colors.red[600],
```

File: lib/theme/index.js [hunk 9]
```diff
@@ -80,6 +80,64 @@ export const generateTheme = ({
...
+        borderColor: colors.red[800],
```
Comment (Betree) [inline lib/theme/index.js]:
```suggestion
        borderColor: colors.red[600],
```

File: lib/theme/index.js [hunk 10]
```diff
@@ -80,6 +80,64 @@ export const generateTheme = ({
...
+        color: 'white',
+      },
+
+      '&:disabled': {
+        backgroundColor: colors.red[50],
+        color: colors.red[200],
```
Comment (Betree) [inline lib/theme/index.js]:
```suggestion
        color: 'white',
```

**Comment by JoyLubega** [comment]
@Betree @flickz I have made the changes

**Comment by flickz** [comment]
@JoyLubega Can you share screenshot of your latest update?

**Comment by JoyLubega** [comment]
Screen Shot 2019-11-28 at 11 28 46 AM
@flickz  I updated the screen shot up with this..

File: components/expenses/MarkExpenseAsUnpaidBtn.js [hunk 11]
```diff
@@ -54,8 +54,12 @@ const MarkExpenseAsUnpaidBtn = ({ id, markExpenseAsUnpaid, refetch }) => {
           </StyledButton>
         </Fragment>
       ) : (
-        <StyledButton onClick={() => setState({ ...state, showProcessorFeeConfirmation: true })} mt={2}>
-          <FormattedMessage id="expense.markAsUnpaid.btn" defaultMessage="Mark as unpaid" />
+        <StyledButton
+          buttonStyle="standard"
```
Comment (Betree) [inline components/expenses/MarkExpenseAsUnpaidBtn.js]: Not a requested change: `standard` is already the default

```suggestion
```

File: components/expenses/MarkExpenseAsUnpaidBtn.js [hunk 12]
```diff
@@ -54,8 +54,12 @@ const MarkExpenseAsUnpaidBtn = ({ id, markExpenseAsUnpaid, refetch }) => {
...
+          onClick={() => setState({ ...state, showProcessorFeeConfirmation: true })}
+          mt={2}
+        >
+          <FormattedMessage id="expense.markAsUnpaid.btn" defaultMessage="Mark As Unpaid" />
```
Comment (Betree) [inline components/expenses/MarkExpenseAsUnpaidBtn.js]: My example in https://github.com/opencollective/opencollective/issues/2570 (`Mark as paid`) was using sentence case, sorry for confusing it by saying title case

```suggestion
          <FormattedMessage id="expense.markAsUnpaid.btn" defaultMessage="Mark as unpaid" />
```

File: components/expenses/PayExpenseBtn.js [hunk 13]
```diff
@@ -127,18 +127,24 @@ class PayExpenseBtn extends React.Component {
...
+        <StyledButton
+          className="pay"
+          buttonStyle="success"
+          onClick={this.onClick}
+          disabled={this.props.disabled || disabled}
+          title={title}
+        >
           {selectedPayoutMethod === 'other' && (
-            <FormattedMessage id="expense.pay.manual.btn" defaultMessage="record as paid" />
+            <FormattedMessage id="expense.pay.manual.btn" defaultMessage="Record As Paid" />
```
Comment (Betree) [inline components/expenses/PayExpenseBtn.js]: My example in https://github.com/opencollective/opencollective/issues/2570 (`Mark as paid`) was using sentence case, sorry for confusing it by saying title case

```suggestion
            <FormattedMessage id="expense.pay.manual.btn" defaultMessage="Record as paid" />
```

File: components/expenses/PayExpenseBtn.js [hunk 14]
```diff
@@ -127,18 +127,24 @@ class PayExpenseBtn extends React.Component {
...
           )}
           {selectedPayoutMethod !== 'other' && (
             <FormattedMessage
               id="expense.pay.btn"
-              defaultMessage="pay with {paymentMethod}"
+              defaultMessage="Pay With {paymentMethod}"
```
Comment (Betree) [inline components/expenses/PayExpenseBtn.js]: My example in https://github.com/opencollective/opencollective/issues/2570 (`Mark as paid`) was using sentence case, sorry for confusing it by saying title case

```suggestion
              defaultMessage="Pay with {paymentMethod}"
```

File: test/cypress/integration/29-host.dashboard.js [hunk 15]
```diff
@@ -56,7 +56,7 @@ describe('host dashboard', () => {
     cy.login({ redirect: '/brusselstogetherasbl/dashboard' });
     cy.get('[data-cy="expense-approved"]').as('currentExpense');
     cy.get('[data-cy="expense-actions"]')
-      .contains('button', 'record as paid')
+      .contains('button', 'Record As Paid')
```
Comment (Betree) [inline test/cypress/integration/29-host.dashboard.js]:
```suggestion
      .contains('button', 'Record as paid')
```

File: test/cypress/integration/29-host.dashboard.js [hunk 16]
```diff
@@ -66,7 +66,7 @@ describe('host dashboard', () => {
     cy.get('[data-cy="expense-paid"]').as('currentExpense');
     cy.get('[data-cy="expense-actions"]')
       .as('currentExpenseActions')
-      .contains('button', 'Mark as unpaid')
+      .contains('button', 'Mark As Unpaid')
```
Comment (Betree) [inline test/cypress/integration/29-host.dashboard.js]:
```suggestion
      .contains('button', 'Mark as unpaid')
```

File: test/cypress/integration/29-host.dashboard.js (hunk 15 above)
Comment (JoyLubega) [reply test/cypress/integration/29-host.dashboard.js]: Done

**Comment by JoyLubega** [comment]
@Betree I have made the changes.  I have also changed  `approved` to Approved

**Pull request merged**